Project.toml

```diff
@@ -9,17 +9,18 @@ FileIO = "5789e2e9-d7fb-5bc7-8068-2c6fae9b9549"
 FixedPointNumbers = "53c48c17-4a7d-5ca2-90c5-79b7896eea93"
 
 [compat]
-ColorTypes = "0.11"
+ColorTypes = "0.11, 0.12"
 FixedPointNumbers = "0.8"
 FileIO = "1.12.0"
 julia = "1.6"
 
 [extras]
 Downloads = "f43a241f-c20a-4ad4-852c-f6b1247861c6"
 ImageIO = "82e4d734-157c-48bb-816b-45c225c6df19"
+ImageMagick = "6218d12a-5da1-5696-b52f-db25d2ecc6d1"
```

Comment:
What's the reason for adding this?

Reply:
When testing in project mode, `FileIO` is complaining if `Imagemagick` is missing.

Reply:
Yucky FileIO... ok.